src/go-postgres/README.md

```diff
@@ -52,7 +52,7 @@ The `ports` property in `docker-compose.yml` [publishes](https://docs.docker.com
 
 ### Installing Go Dependencies
 
-This template includes the popular [PostGres Driver Library for Go](github.com/lib/pq). This is the recommended driver for use with Go, as per [GoLang Documentation](https://golangdocs.com/golang-postgresql-example).
+This template includes the popular [PostGres Driver Library for Go](https://github.com/lib/pq). This is the recommended driver for use with Go, as per [GoLang Documentation](https://golangdocs.com/golang-postgresql-example).
```

Comment:
Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file and update NOTES.md instead to avoid merge conflicts? Thanks!

Reply:
Thanks, I didn't realize this, so I've made the suggested changes.